core/templates/dev/head/collection_editor/collection_details_directive.html

```diff
@@ -0,0 +1,17 @@
+<script type="text/ng-template" id="inline/collection_details_directive">
+  <p>title: <[getTitle()]> </p>
+  <p>category: <[getCategory()]> </p>
+  <p>objective: <[getObjective()]> </p>
+  <p>version: <em><[getVersion()]></em></p>
+
+  <br>
+  <input ng-model="newExpId" type="text" placeholder="New exp ID" /><button ng-click="addExploration(newExpId)">Add Exploration</button>
```

Comment:
Nit: don't need /> for input tags. Can follow the style for the other inputs below.

Also, s/exp/exploration in the placeholder.